scripts/sign_all.py

```diff
@@ -1,43 +0,0 @@
-#!/usr/bin/env python
```

Comment:
I was going to fix the module-level imports not at the top of the file, and then realized that it's been replaced by https://github.com/mozilla/olympia/blob/master/apps/addons/management/commands/process_addons.py#L30 (to sign all addons) and https://github.com/mozilla/olympia/blob/master/apps/addons/management/commands/sign_addons.py for specific addons.